doc/application/index.rst

```diff
@@ -162,10 +162,24 @@ subdirectories which are not described here.
     package.
 
 
+Example Application
```

Comment:
I think we should make it clear that this example application is based on a manifests, it is not your typical application that we have across the tree without modules. We do not want to have people go through the trouble of setting up something like for a basic application that does not require any of the items listed below.

Reply:
I agree. Maybe something like "Example standalone application" and then specifically mention west and the [T2 topology](https://docs.zephyrproject.org/latest/guides/west/workspaces.html#topologies-supported) specifically.